client/my-sites/customer-home/cards/education/quick-start-video/index.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { Card } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import InlineSupportLink from 'components/inline-support-link';
+import { localizeUrl } from 'lib/i18n-utils';
+// import isUnlaunchedSite from 'state/selectors/is-unlaunched-site';
+//import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+
+/**
+ * Style dependencies
+ */
+import quickStartVideoImage from 'assets/images/customer-home/quick-start-video-ss.png';
+import './style.scss';
+
+export const QuickStartVideo = () => {
```

Comment:
Since this is outside of Learn and Grow, we'll need to add a similar event to https://github.com/Automattic/wp-calypso/pull/44266/files#diff-f718bf14b2a68d4096184913827aca05R76 if we'd like impression tracking.

Reply:
🤔 I might have missed the stats card impression now that I think of it.